integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -69,6 +72,13 @@
 
   private static Map<Integer, Expression> filterMap = new HashMap<>();
 
+  private final static String HIVE_DEFAULT_DYNAMIC_PARTITION = "__HIVE_DEFAULT_PARTITION__";
```

Comment:
can you explain, why need to add "HIVE_DEFAULT_DYNAMIC_PARTITION" ? what purpose?
if (value == null) {
        filter.add(carbonDataColumnHandle.getColumnName() + "=" + HIVE_DEFAULT_DYNAMIC_PARTITION);

Reply:
HIVE_DEFAULT_DYNAMIC_PARTITION is created when the partition value for that column is null so the data goes to __HIVE_DEFAULT_PARTITION__ partition. This check is added so that we can query the data from __HIVE_DEFAULT_PARTITION__ as well.

Reply:
ok